spring/src/main/java/org/axonframework/spring/eventsourcing/SpringRepositoryFactoryBean.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.spring.eventsourcing;
```

Comment:
The `Repository` is not an Event Sourcing _only_ component, as a user might just as well have a state-stored aggregate.
As such, we should move this class to another directory, e.g. `modelling`.